source/auth/auth.rst

```diff
@@ -232,7 +232,11 @@ MONGODB-X509
 
 :since: 2.6
 
-MONGODB-X509 is the usage of X-509 certificates to validate a client.  The server will use the distinguished subject name of the client certificate in the SSL negotiation to authenticate. The driver will be required to supply the distinguished subject name outside of the SSL negotiation to the server using the "authenticate" command.
+MONGODB-X509 is the usage of X-509 certificates to validate a client.  The server will use the distinguished subject name of the client certificate in the SSL negotiation to authenticate. The driver MAY supply the distinguished subject name outside of the SSL negotiation to the server using the "authenticate" command.
```

Comment:
MAY -> MUST. The changes in MongoDB 3.4 don't change the requirement for earlier server versions. You're probably better off rewriting this whole section, separating the issue of when the application must provide the subject name from the issue of when the driver has to provide the subject name to the authenticate command. They are two separate issues.